src/packs/abilities/Fury_NsE50OrdFsIk4ND2/Level_1_9cPQyQCXUIXRme9R/5_Ferocity_9iT8OXz2ZLsuyduZ/ability_To_the_Uttermost_End_54IP1nIQiFvWmqYw.json

```
@@ -0,0 +1,86 @@
+{
+  "folder": "9iT8OXz2ZLsuyduZ",
+  "name": "To the Uttermost End",
+  "type": "ability",
+  "_id": "54IP1nIQiFvWmqYw",
+  "img": "icons/magic/unholy/strike-body-explode-disintegrate.webp",
+  "system": {
+    "source": {
+      "book": "Heroes",
+      "page": "134",
+      "license": "Draw Steel Creator License",
+      "revision": 1
+    },
+    "_dsid": "to-the-uttermost-end",
+    "story": "You gut your life force to ensure a foe’s demise.",
+    "keywords": [
+      "melee",
+      "strike",
+      "weapon"
+    ],
+    "type": "main",
+    "category": "heroic",
+    "resource": 5,
+    "trigger": "",
+    "distance": {
+      "type": "melee",
+      "primary": 1
+    },
+    "damageDisplay": "melee",
+    "target": {
+      "type": "creature",
+      "value": 1
+    },
+    "power": {
+      "roll": {
+        "formula": "@chr",
+        "characteristics": []
+      },
+      "effects": {
+        "UZzL5kwla7Qtvarz": {
+          "name": "Damage",
+          "img": null,
+          "type": "damage",
+          "_id": "UZzL5kwla7Qtvarz",
+          "damage": {
+            "tier1": {
+              "value": "7 + @chr"
+            },
+            "tier2": {
+              "value": "11 + @chr"
+            },
+            "tier3": {
+              "value": "16 + @chr"
+            }
+          }
+        }
+      }
+    },
+    "effect": {
+      "before": "",
+      "after": ""
+    },
+    "spend": {
+      "text": "While you are winded, this ability deals an extra  1d6 damage for each ferocity spent. While you are dying, it deals an  extra 1d10 damage for each ferocity spent. In either case, you lose  1d6 Stamina after making this strike.",
+      "value": 1
```

```suggestion
      "value": null
```

Variable spends you actually leave blank